rotkehlchen/exchanges/exchange.py

```diff
@@ -352,18 +352,22 @@ def send_unknown_asset_message(
             self,
             asset_identifier: str,
             details: str,
+            location: Location | None = None,
     ) -> None:
         """Log warning and send WS message to notify user of unknown asset found on an exchange.
         Args:
             asset_identifier (str): Asset identifier of the unknown asset.
             details (str): Details about what type of event was being processed
                 when the unknown asset was encountered.
+            location (Location | None): Location of the exchange where the unknown asset was found.
+                Uses self.location if None. Used for some exchanges that have multiple locations
+                but common asset mappings like Coinbase and Coinbase Prime. Defaults to None.
```

Comment:
why do it like this and not just keep it as it was and just do an override in coinbaseprime and binance? This way you avoid the if check in all of them.